paddle/fluid/operators/run_program_op.h

```diff
@@ -300,24 +300,16 @@ class RunProgramGradOpKernel : public framework::OpKernel<T> {
 
     // Step 3. run ops
     exe.RunPartialPreparedContext(exe_ctx.get(), &scope, start_op_index,
-                                  end_op_index, false, true, true);
+                                  end_op_index, /*create_local_scope=*/false,
+                                  /*create_vars=*/true, /*keep_kids=*/true);
 
     // Step 4. copy outputs
-    // TODO(chenweihang): need copy?
+    // NOTE(chenweihang): can't remove this copy when gradient is SelectedRows
```

Comment:
I forgot to change this line comment. If SelectedRows is converted to LoDTensor inside op, need to copy

Reply:
done